CMakeLists.txt

```diff
@@ -188,6 +188,8 @@ else()
 endif()
 precice_validate_prettyprint()
 
+# libfmt
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/fmt)
```

Comment:
That's only for the libraray. You need to add an include directory for the binary and tests as well in order to compile
```cmake 
  include_directories(${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/fmt/include)
```

Reply:
This processes the [`CMakeLists.txt`](https://github.com/precice/precice/blob/b4aa52aca2c086782d02badc1c60955ba5593cf4/thirdparty/fmt/CMakeLists.txt) in the `thirdparty/fmt` directory.

Reply:
Still, it didn't compile on system without adding `include_directories`. Have you enabled testing and the binary building?

Reply:
I tested it what is the error message?

The include directory is propagated by the target.

Reply:
Tested it now as well in a fresh build and it works for me as well. I previously rebuild an old build, but still resourced the root CMakeLists.txt in order to incorporate the new include directory. I have no idea why. Sorry for the mess.